platform.md

```diff
@@ -157,6 +157,18 @@ The stack ID:
 - MUST only contain numbers, letters, and the characters `.`, `/`, and `-`.
 - SHOULD use reverse domain name notation to avoid name collisions.
 
+### Stack Validation
+
+During the prepare phase, the stack properties of the new run-image MUST be compared to the stack properties of the previous run-image. If any of the following are true, the analysis MUST fail:
```

Comment:
> analysis MUST fail

Is analysis right here? Or should it be updated to be say `prepare` or similar?

Reply:
Fixed in 47a53f3